src/components/ReceiveMyAddress.js

```diff
@@ -16,14 +16,16 @@ import { useDispatch, useSelector } from 'react-redux';
 import SimpleButton from './SimpleButton';
 import CopyClipboard from './CopyClipboard';
 import { sharedAddressUpdate } from '../actions';
+import { COLORS } from '../styles/themes';
 
 export default function ReceiveMyAddress() {
   const dispatch = useDispatch();
   const wallet = useSelector((state) => state.wallet);
   const lastSharedAddress = useSelector((state) => state.lastSharedAddress);
 
   const getNextAddress = async () => {
-    const { address, index } = await wallet.getNextAddress();
+    // Fetch the address but do not mark it as used yet.
+    const { address, index } = await wallet.getCurrentAddress();
```

Comment:
What's this change? Isn't this PR just for color changes? 

And I think this is not correct, we should get the next address, not current.

Reply:
This escaped from the latest rebase with `dev`, thanks for pointing out!
Fixed on [e18e360](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/327/commits/e18e3609ae83e87c8081a4c265db2e93276965be).